backup.sh

```diff
@@ -193,7 +198,7 @@ delete-thinning () {
   fi
 
   local CURRENT_INDEX=0
-  local BACKUPS=($(ls -r $BACKUP_DIRECTORY)) # List newest first
+  local BACKUPS=($(ls -r $BACKUP_DIRECTORY | grep $WORLD_NAME)) # List newest first
```

Comment:
Is there a more robust way of filtering backups for a particular world? I can see this going wrong where one world is called `world` and another world is called `world_nether`. When `$WORLD_NAME="world"` grep will also return results for `world_nether` since `world` is a substring of `world_nether`.

Reply:
```suggestion
  local BACKUPS=($(ls -r $BACKUP_DIRECTORY | grep "$WORLD_NAME$")) # List newest first
```

Wouldn't this be sufficient @nicolaschan ?
Matches to the end of the string so `world_nether` should not be matching on `world`

Reply:
Nvm a bit more stuff to do, does not look like this was tested beforehand at all